### PR TITLE
fix(bazel): change structure in `.template` files for correct start work with `ng serve` command.

### PR DESCRIPTION
**Pull request by DeveloperFromUkraine**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #29726

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information

### CODE REVIEW

File: packages/bazel/src/builders/files/WORKSPACE.template [hunk 1]
```diff
@@ -34,7 +34,7 @@ http_archive(
 # Load and install our dependencies downloaded above.
 
 load("@build_bazel_rules_nodejs//:defs.bzl", "check_bazel_version", "node_repositories", "yarn_install")
-check_bazel_version("0.22.0")
+check_bazel_version("0.24.0")
```
Comment (alexeagle) [inline packages/bazel/src/builders/files/WORKSPACE.template]: This should be the minimum version. Is there a reason that 0.22.0 doesn't work anymore?

File: packages/bazel/src/builders/files/src/BUILD.bazel.template [hunk 2]
```diff
@@ -77,20 +77,28 @@ filegroup(
...
     ],
     index_html = "index.html",
-    deps = [":src"],
+    deps = [
+        ":src",
+        ":rxjs_umd_modules",
+        "@npm//node_modules/tslib:tslib.js",
+
+        "@npm//node_modules/@angular/common:bundles/common.umd.js",
+        "@npm//node_modules/@angular/common:bundles/common-http.umd.js",
```
Comment (alexeagle) [inline packages/bazel/src/builders/files/src/BUILD.bazel.template]: We should not need to add these files here. Is this a rebase mistake?

File: packages/bazel/src/builders/files/WORKSPACE.template (hunk 1 above)
Comment (DeveloperFromUkraine) [reply packages/bazel/src/builders/files/WORKSPACE.template]: No, I just added a new version of Bazel to be able to use the most recent changes.

File: packages/bazel/src/builders/files/src/BUILD.bazel.template (hunk 2 above)
Comment (DeveloperFromUkraine) [reply packages/bazel/src/builders/files/src/BUILD.bazel.template]: No, this is not a rebase mistake. When I tried to run `ng build --leaveBazelFilesOnDisk` it's failed. And I managed to fix it thanks to these manipulations.

Comment (alexeagle) [reply packages/bazel/src/builders/files/src/BUILD.bazel.template]: in that case, the bug is somewhere else, and this is just papering over it.
Is there an issue filed that describes the problem and how to reproduce?

Comment (DeveloperFromUkraine) [reply packages/bazel/src/builders/files/src/BUILD.bazel.template]: Yes this is how, now we could resolve that issue. And then try to find real error.

Issue: https://github.com/angular/angular/issues/29726

**Comment by AndrewKushnir** [comment]
Closing this PR since the experimental Bazel builder support in Angular CLI was deprecated in Angular v10.

**Pull request closed without merging**